app/classifier/classifier.jsx

```diff
@@ -267,7 +279,7 @@ class Classifier extends React.Component {
               toggleExpertClassification={this.toggleExpertClassification}
             />
           }
-
+          <ModelScore workflow={this.props.workflow} />
```

Comment:
Does this component need the entire workflow object if all it does is render a score?

Reply:
the workflow was being passed as it contains information used by the wrapper inside ModelScore to check if it's a "modelling workflow".

Reply:
Maybe pass that in as a `modellingEnabled` prop instead? Similar to how pan-zoom is handled: https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/components/frame-viewer.jsx#L59-L65

It would keep all the complicated conditional code in one place by checking once and passing the boolean result down. I think feedback does something similar.